docs/kyma/docs/017-details-local-installation.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Local installation
+type: Details
+---
+
+Simple as it may seem, the Kyma installation is a complex and dynamic process. This document extends the “031-gs-local-installation.md” guide with a detailed breakdown of the alternative installation method, i.e. the “run.sh” script.
+
+> **NOTE:** the ‘run.sh’ script should be used exclusively for development purposes.
+
+To start the local installation, run the following command:
+```
+./installation/cmd/run.sh
+```
+
+This script sets up default parameters, starts Minikube, builds Kyma-Installer, generates local configuration, creates the `installation` Custom Resource and sets up Installer. Subsequent sections provide a detailed description of each step.
```

Comment:
- `the Kyma-Installer` -> `the` is missing here
- We have decided within the TW team that we will format the resources this way: `the Installation custom resource` -> would that be ok to update that in the whole doc? Apologies for a different approach during the first review...
- `resource, and sets up`-> serial comma was missing

Reply:
shouldn't it be `the Installation Custom Resource`? The rest was corrected